Felienne/hedy
# Remove /fix-code endpoint #2435

**Please fill out this template by replacing all content between < >**

< Remove /fix-code endpoint >

**Description**

< Remove /fix-code endpoint > Use present tense without a subject to describe your PR, for example: "Adds translations of levels 1 to 12 to Polish"

**Fixes < #2435 >**

< Always link the number of the issue or of the discussion that your pr concerns. >
Notable exception: adding content may be done without an accompanying issue 

**How to test**

Run Hedy locally and go to level 1. Execute the following programs:
- ```aks something``` the program should result in an error. There should be a light bulb and when you click it, it should display the fixed code ```ask something```
- ```   print something``` the program should result in a warning. Again, there should be a light bulb and on-click, it should display the fixed code without the leading whitespaces
- ```turn bla``` should result in an error. Note that there should be no light bulb.

**Checklist**
Done? Check if you have it all in place using this list*
  

* If you're unsure about any of these, don't hesitate to ask. We're here to help!
